Lombiq.Tests.UI/Services/SqlServerManager.cs

```diff
@@ -153,7 +174,10 @@ public SqlServerRunningContext CreateDatabase()
         }
     }
 
-    public void RestoreSnapshot(string snapshotDirectoryPath)
+    public async Task RestoreSnapshotAsync(
```

Comment:
Try a failing SQL Server test to see if failure dump creation works properly too.

Reply:
Works, here: https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/actions/runs/2325275960